src/UNL/UCBCN/Calendars.php

```diff
@@ -0,0 +1,39 @@
+<?php
+namespace UNL\UCBCN;
+
+use UNL\UCBCN\ActiveRecord\RecordList;
+use UNL\UCBCN\ActiveRecord\Record;
+/**
+ * Object related to a list of calendars.
+ * 
+ * PHP version 5
+ * 
+ * @category  Events 
+ * @package   UNL_UCBCN
+ * @author    Tyler Lemburg <trlemburg@gmail.com>
+ * @copyright 2015 Regents of the University of Nebraska
+ * @license   http://www1.unl.edu/wdn/wiki/Software_License BSD License
+ * @link      http://code.google.com/p/unl-event-publisher/
+ */
+
+class Calendars extends RecordList
+{
+    public function getDefaultOptions() {
+        return array(
+            'listClass' =>  __CLASS__,
+            'itemClass' => __NAMESPACE__ . '\\Calendar',
+        );
+    }
+
+    public function getSQL() {
+        if (array_key_exists('account_id', $this->options)) {
```

Comment:
I think `isset($this->options['account_id'])` is easier to read, probably just because I'm used to it.  I think it is also quicker to write and I've heard it is quicker than using `array_key_exists`, tho they do check slightly different things.  What do you think?

http://stackoverflow.com/questions/3210935/difference-between-isset-and-array-key-exists